src/components/Main/Main.html

```diff
@@ -0,0 +1,32 @@
+<aside class="sidebar mdc-drawer mdc-drawer--dismissible">
```

Comment:
is it possible to wrap this all inside a div?

Reply:
Nope.... unfortunately no. This will not work

Reply:
I have tried to do this and everything seems to work as before

![image](https://user-images.githubusercontent.com/43406825/47569315-f87c6580-d93b-11e8-830d-08dce2d7a6ab.png)

Reply:
It is possible to wrap both <aside> and the following <div> https://screencast.com/t/QOHFKyQvAAM
Is this something you want me to do?

If I wrap <aside> only then I will have some bugs.

Do you want to change the structure, and to create separate component "Sidebar"? I will mount it within Main component? In this case I will need mount Header within a sidebar, since in MDC they are working together and header is wrapped by special sidebar div.

Just in case - I am in our training room. Jump in if you around :-)